Docs/Debugging/Keil.md

```diff
@@ -1,122 +0,0 @@
-#CMSIS DAP MDK
```

Comment:
@AnotherButler I'm a bit unsure about this. The information is superseded by the info in 'Debugging with uVision' (upgraded to uVision 5), but the old Keil.md looks a bit more information dense, I'm wondering if we need to explain to people what a breakpoint is... Especially if they just shelled out XXXX for a uVision license.